entity-types/uninstrumented-awsdynamodbtable/definition.yml

```diff
@@ -0,0 +1,6 @@
+domain: UNINSTRUMENTED
+type: AWSDYNAMODBTABLE
+
+configuration:
+  entityExpirationTime: DAILY
+  alertable: false
```

Comment:
why would a dynamodb not be alertable?

Reply:
Because, as all uninstrumented entities, this is an entity which reports no data at all.